test/rhsmlib_test/base.py

```diff
@@ -71,6 +71,7 @@ def injection_definitions(self, *args, **kwargs):
         raise NotImplementedError("Subclasses should define injected objects")
 
 
+@unittest.skip("D-Bus tests are not reliable.")
```

Comment:
i think a slightly better way can be to tag the tests as such, ie:
```python
from nose.plugins.attrib import attr

@attr('dbus')
class DBusObjectTest(unittest.TestCase):
```
and then adding `!dbus` to the `attr` of the nosetest config file